src/vue/modules/assets/asset-explorer/components/asset-actions-bar.spec.js

```diff
@@ -0,0 +1,135 @@
+import AssetActionsBar from './asset-actions-bar'
+
+import Vuex from 'vuex'
+
+import { assetExplorerModule } from '../store/index'
+import { Asset } from '../../shared/wrappers/asset'
+
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+
+import { Bus } from '@/js/helpers/event-bus'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+
+const localVue = createLocalVue()
+localVue.use(Vuex)
+
+describe('Asset explorer module', () => {
+  const props = {
+    asset: new Asset({
+      id: 'USD',
+    }),
+  }
+
+  let store
+  let wrapper
+
+  beforeEach(() => {
+    store = new Vuex.Store({
+      modules: { 'asset-explorer': assetExplorerModule },
+    })
+
+    wrapper = shallowMount(AssetActionsBar, {
+      store,
+      localVue,
+      propsData: props,
+    })
+  })
+
+  describe('method', () => {
+    describe('addBalance', () => {
+      beforeEach(() => {
+        sinon.stub(wrapper.vm, 'createBalance').resolves()
+        sinon.stub(Bus, 'success')
+        sinon.stub(ErrorHandler, 'process')
+      })
```

Comment:
these stubs should not be generalized. they actually needed in couple of tests, stub them only where needed

Reply:
Updated